### PR TITLE
Reorganize docs

### PR DESCRIPTION
**Pull request by JuanitoFatas**
## Summary
- Split into WINDOWS & OS X  & LINUX & INSTRUCTORS
- Extract troubleshooting into TROUBLESHOOTING.md
## File tree

```
.
├── README-LINUX.md
├── README-OSX.md
├── README-WINDOWS.md
├── README.md
├── TROUBLESHOOTING.md
├── Vagrantfile
├── images
│   ├── git-001.png
│   ├── git-002.png
│   ├── git-003.png
│   └── welcome.png
├── instructor
│   └── README.md
├── sail.sh
└── sample.log
```
## Preview
- [README](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/README.md)
  - [README-WINDOWS.md](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/README-WINDOWS.md)
  - [README-OSX.md](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/README-OSX.md)
  - [README-LINUX.md](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/README-LINUX.md)
- [TROUBLESHOOTING](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/TROUBLESHOOTING.md)
- [README for Instructor](https://github.com/jollygoodcode/rails-carrier/blob/windows-and-os-x/instructor/README.md)

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@winston :eyes:

File: README-WINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+> _This should take about 10 min or less._ [See sample log](sample.log).
+
+Open up your Terminal.
+
+click on Windows Start and type "cmd" in the "Search programs and files" search field.
+
+Open `cmd.exe`. That's the Terminal equivalent in Windows.
+
+Create and change into into a directory which you'll be using for all of your coding needs, e.g. `~/workshop/` or `C:\workshop`:
```
Comment (winston) [inline README-WINDOWS.md]: Create and change into into a directory which you'll be using for all of your coding needs, e.g. `C:\workshop`:

File: README-WINDOWS.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+```
+$> mkdir workshop
```
Comment (winston) [inline README-WINDOWS.md:66]: Windows equivalent is `MD workshop` is it?

File: README-WINDOWS.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+$> cd workshop
+```
+
+Download our pre-built _Linux_ (Vagrant) box - Ubuntu 14.04, Ruby 2.3.0 and Rails 4.2.5.1.
+
+[First check you're using 32-bit / 64-bit machine](https://support.microsoft.com/zh-tw/kb/827218).
```
Comment (winston) [inline README-WINDOWS.md]: First check if you're using 32-bit or 64-bit machine. Otherwise, just follow the instructions for a 32-bit machine.

**Comment by winston** [comment]
Awesome! :bow:

File: README-WINDOWS.md (hunk 2 above)
Comment (JuanitoFatas) [inline README-WINDOWS.md:66]: mkdir also works. https://technet.microsoft.com/library/bb490930.aspx

Comment (winston) [inline README-WINDOWS.md:66]: ok!

**Pull request merged**